INSTALL.md

```diff
@@ -348,6 +349,26 @@ https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/services/misc/matrix-
 
 Once you have installed synapse as above, you will need to configure it.
 
+
+## Setting your server_name
```

Comment:
The instructions above talk about generating the config and setting the server_name, so this is too late.

Reply:
basically it feels like this section isn't adding anything here. One idea would be to have a section which explains what the server_name does, but instructions here on how you change this one configuration option don't seem to be very helpful.

Reply:
Agreed, it shouldn't be here - have deleted since it doesn't add much over the config at the beginning of the file